event-app-ui/src/components/App/App.jsx

```diff
@@ -0,0 +1,14 @@
+import "./App.css";
+import LoginForm from "../Login/LoginForm";
+import SignUpForm from "../SignUpForm/SignUpForm";
+import Home from "../Home/Home";
```

Comment:
Please use absolute imports instead of relative imports https://plainenglish.io/blog/why-and-how-to-use-absolute-imports-in-react